client/src/styles/components/_chats-list.scss

```diff
@@ -1,103 +1,148 @@
-    font-size: 1.5rem;
-    font-weight: 500;
-    margin-bottom: 1.75rem;
-    font-family: $font-secondary;
-  }
-
-  .list {
+  &>.content {
+    position: fixed;
+    width: 100vw;
+    height: 65vh;
+    overflow-x: hidden;    
     list-style: none;
     padding: 0;
     margin: 0;
-    display: flex;
-    flex-direction: column;
     gap: 0.875rem;
-    flex: 1;
     overflow-y: auto;
     -ms-overflow-style: none;
     scrollbar-width: none;
 
     &::-webkit-scrollbar {
       display: none;
     }
+  }
 
-    .chats-ended {
-      margin: 0.625rem 0 -0.5rem;
-      color: #7f84a3;
-      font-size: 0.875rem;
-      font-weight: 500;
-      mix-blend-mode: multiply;
+  h1 {
+    margin: 3rem 2rem 2rem 0;
+    color: $default-text-color;
+    font-family: $font-secondary;
+    font-weight: 500;
+    font-size: 1.5rem;
+  }
+
+  h2 {
+    margin: 1rem 2rem 0 0;
+    color: $ended-chats-color;
+    font-weight: 500;
+    font-size: 1rem;
+  }
+
+  .loading {
+    color: $default-text-color;
+    margin-right: 2rem;
+  }
+
+  .chat-item {
+    background: rgba($color: white, $alpha: 0.8);
+    padding: 0.75rem;
+    margin: 0 9vw 1rem 9vw;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    column-gap: 0;
+    outline: none;
+    cursor: pointer;
+
+    &:focus,
+    &:active {
+      border: $chat-item-border;
     }
 
-    .chat-item {
-      height: 3.25rem;
+    .content {
       display: flex;
-      justify-content: space-between;
-      padding: 0 1rem;
-      align-items: center;
-      border-radius: 8px;
-      border: 2px solid transparent;
-      background: rgba(255, 255, 255, 0.5);
-      outline: none;
-      cursor: pointer;
-
-      &:focus,
-      &:active {
-        border: 2px solid #736cff;
-      }
+      flex-direction: column;
+      line-height: 1.1rem;
+      color: $sub-text-color;
 
       .name {
-        color: #0e1c74;
+        color: $default-text-color;
         font-size: 0.875rem;
         font-weight: 600;
       }
 
-      &-ended {
-        .name {
-          color: #848484;
-        }
-
-        .arrow path {
-          stroke: #304571;
-        }
+      .last-message {
+        font-weight: 300;
       }
     }
+
+    .rest {
+      display: flex;
+      align-items: center;
+    }
+
+    .notification {
+      display: flex;
+      position: relative;
+    }
+
+    .unread-messages {
+      color: white;
+      font-weight: bolder;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+    }
+
+    .arrow path {
+      stroke: #304571;
+    }
+  }
+
+  .ended-chats .content .name {
+    color: $chat-sub-title-color;
   }
 
-  .new-supportee {
-    margin: auto 0 1.25rem;
-    padding-top: 1.75rem;
+  .footer {
+    box-shadow: 0px -20px 35px $dark-purple;
+    background-color: $dark-purple;
+    position: fixed;
+    bottom: 0;
+    width: 100vw;
+    height: 18vh;
```

Comment:
Same as before. Just making sure that it's understandable that I'm talking about the specific values and not the 100vw/vh - which is okay